### PR TITLE
Updates to Uncrustify Formatting

### PR DESCRIPTION
**Pull request by Skptak**
Splitting up #68 into more manageable single PRs containing just the changes to the individual actions
Will not be doing the YML files one by one, so please reference the #68 actions for proof of working

Relevant test logs can be found [here](https://github.com/Skptak/CI-CD-Github-Actions/actions/runs/5870035225/job/15916258663)

### CODE REVIEW

**Comment by kstribrnAmzn** [comment]
Does this not duplicate work from https://github.com/FreeRTOS/CI-CD-Github-Actions/pull/82?

**Comment by Skptak** [comment]
> 

This is for uncrustify formatting

**Review by rawalexe — APPROVED** [review]

**Review by rawalexe — APPROVED** [review]

File: .github/workflows/formattingTests.yml [hunk 1]
```diff
@@ -0,0 +1,295 @@
+name: Formatting Tests
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [main,v2]
```
Comment (paulbartell) [inline .github/workflows/formattingTests.yml:7]: will this branch always exist?

File: formatting/goodFiles/source/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#include <stdint.h>
```
Comment (paulbartell) [inline formatting/goodFiles/source/README.md:1]: Is this supposed to be named README.md?

**Review by paulbartell — APPROVED** [review]
It's not super clear from first glance what each of the test files is verifying. It would be preferable to specify what rules / portion of the style standard are being validated by each test file.

This is better than the current state of this check, so approving it.

**Pull request merged**